test/functional/Progress.hs

```diff
@@ -33,7 +33,7 @@ tests =
                 let path = "diagnostics" </> "Foo.hs"
                 _ <- openDoc path "haskell"
                 expectProgressMessages [pack ("Setting up testdata (for " ++ path ++ ")"), "Processing", "Indexing"] []
-        , requiresEvalPlugin $ testCase "eval plugin sends progress reports" $
+        ,  knownBrokenForGhcVersions [GHC92] "No evaluation status with GHC 9.2" $ requiresEvalPlugin $ testCase "eval plugin sends progress reports" $
```

Comment:
Any idea why this is broken?

Reply:
Unfortunately no.

Reply:
I'd put that in a comment! It's always useful to know the cause, and it's also useful to know "it's a mystery"!

Reply:
Do you remember what it fails with? Whether it hangs, or says "unexpected <something>", and whether it fails consistently?
I remember fixing a non-determinstic fail here so I can probably look at this later.

Reply:
It consistently returns the final result, without any progress step.